src/pubtools/sign/utils.py

```diff
@@ -1,4 +1,8 @@
 import datetime
+import subprocess
+import os
+
+from .conf.conf import CONFIG_PATHS
```

Comment:
Is seems a little odd to hard code the paths. Perhaps add support to set it by an environment variable as well?

Reply:
It's set to standard paths which are usually used. I don't think it will change every, obliviously you can load your own config file with --config